pom.xml

```diff
@@ -68,7 +71,13 @@
 		<dependency>
 			<groupId>com.google.code.gson</groupId>
 			<artifactId>gson</artifactId>
-			<version>2.9.0</version>
+			<version>2.11.0</version>
+			<exclusions>
```

Comment:
Need to be replaced by the API plugin https://plugins.jenkins.io/gson-api/

Reply:
Thank you @jonesbusy it is done now